birdhouse/README.rst

```diff
@@ -15,8 +15,15 @@ Requirements
 * User running ``BIRDHOUSE_COMPOSE`` below must not be ``root`` but a regular user
   belonging to the ``docker`` group.
 
-* Install latest docker-ce and docker-compose for the chosen distro (not the
-  version from the distro).
+* `Install latest version of docker <https://docs.docker.com/engine/install/>`_ for the chosen distro 
+   (not the version from the distro).
+
+  * Please ensure the latest versions of the following packages and any of their dependencies
+    are installed for your distro:
+
+    * docker engine
+    * docker CLI
+    * docker compose plugin (v2.20.2+ is required)
```

Comment:
Good idea to have this explicitly. 
Are there any specific docker engine/CLI versions required to work with compose `2.20.2+`?

Reply:
Found something... 

```
docker-compose-plugin is already the newest version (2.18.1-1~ubuntu.18.04~bionic).
```

This change enforces Ubuntu 20.04+

Reply:
I think I'm OK with that. 18.04 was EOL in 2023. I'll update the README.

Reply:
>  Are there any specific docker engine/CLI versions required to work with compose 2.20.2+?

I tried to find any documentation about that but I wasn't able to find anything specific. Certain versions of Docker Desktop come with specific versions of docker compose but as far as the engine/CLI I can't find anything about specific version compatibilities.

I'm just assuming that if you have a relatively modern version of docker engine you'll be compatible with a relatively modern docker compose. :shrug: